Lib/fontTools/help.py

```diff
@@ -0,0 +1,51 @@
+import pkgutil
+import sys
+import fontTools
+import importlib
+import os
+from pathlib import Path
+
+def _discover_path_importables(pkg_pth, pkg_name):
+    """Yield all importables under a given path and package."""
```

Comment:
I don't think you need this `_discover_path_importables ` function, you can just call `pkgutil.walk_packages` directly (it works recursively)

```python
sorted(mod.name for mod in pkgutil.walk_packages([fontTools.__path__[0]], prefix="fontTools."))
```